README.md

```diff
@@ -24,25 +17,89 @@ Change directory to the package under development. Note: an integration is a spe
 
 ```bash
 cd integrations
-cd package/my-integration
+cd package/my-package
 ```
 
-Run the `help` command and see available actions:
+Run the `help` command and see available commands:
 
 ```bash
 elastic-package help
 ```
 
-## GitHub authorization
+## Commands
```

Comment:
Did you miss the test command or is it intended to cover it later on once system tests are done?

Reply:
I intended to add it when system tests are done but, now that you mention it, we have pipeline tests so maybe we should add it to the README now itself? Or would you prefer to wait until we have defined pipeline test files in a couple of integrations packages first?

Reply:
I'm fine with both options, we can wait with announcing them officially.

Reply:
I'm going to wait to add the `test` command here until https://github.com/elastic/integrations/pull/262 is merged. We can easily update the README at that time.

Reply:
FYI PR is merged.

Reply:
Great! I will add `test` here now.

Reply:
Added a section in bd93e6a. 

But I think we need to now define what a Pipeline Test is and how a package author can set it up. Let's do this in a follow up PR. I added a checklist item for this in https://github.com/elastic/elastic-package/issues/75. @mtojek Since you worked on the pipeline tests, can you take on this follow up PR?

Reply:
HOWTO guides is a good idea. I can write some docs about it.